test/helpers/fixtures.js

```diff
@@ -1,22 +1,29 @@
 import db from '../../src/services/db';
-import * as baseSql from '../../src/base/sql';
+
+import pgp from '../../src/base/pgp';
 
 const data = {
   state: require('../fixtures/state')
 };
 
+function resetSequence () {
+  return Promise.all(Object.keys(data).map(table => {
+    return db.task(t => {
+      const tableName = new pgp.helpers.TableName(table);
+      return t.one('SELECT MAX(id) FROM $1', tableName).then(result => {
```

Comment:
For reference, I know how I got to my original spot: `db.func` in `pg-promise` doesn't support that kind of syntax. I had to do a direct query (and not use `func`) to get this to work.